.travis.yml

```diff
@@ -22,11 +22,14 @@ install:
   - "[ \"$NUMPY\" = latest ] && pip install --upgrade numpy || [ -z \"$NUMPY\" ] || pip install numpy==$NUMPY"
   - "[ \"$PANDAS\" = latest ] && pip install --upgrade pandas || [ -z \"$PANDAS\" ] || pip install pandas==$PANDAS"
   - "[ \"$SCIKITLEARN\" = latest ] && pip install --upgrade scikit-learn || [ -z \"$SCIKITLEARN\" ] || pip install scikit-learn==$SCIKITLEARN"
+  - "[ \"$STATSMODELS\" = latest ] && pip install --upgrade statsmodels || [ -z \"$STATSMODELS\" ] || pip install statsmodels==$STATSMODELS"
   - "[ \"$DASK\" = latest ] && pip install --upgrade dask || [ -z \"$DASK\" ] || pip install dask==$DASK"
   - "[ \"$DISTRIBUTED\" = latest ] && pip install --upgrade distributed || [ -z \"$DISTRIBUTED\" ] || pip install distributed==$DISTRIBUTED"
   # need to downgrade tornado manually
   - "[ \"$SCIPY\" = latest ] || [ -z \"$SCIPY\" ] || pip install tornado==4.5.3"
   - "[ \"$SCIPY\" = latest ] && pip install --upgrade scipy || [ -z \"$SCIPY\" ] || pip install scipy==$SCIPY"
+  # recent dask requires fsspec to load data from disk
+  - "[ \"$DASK\" = latest ] && pip install fsspec || true"
```

Comment:
I would actually prefer to have this in the test-requirements.txt, don't you think?

Reply:
Yes, this will work since fsspec is provided as a conda package and its only requirement is Python >= 3.5.  But I am not sure this is a good idea, you do not need fsspec if you have an older dask.
Anyway I have no strong feeling, feel free to change it if you prefer.

Reply:
Thinking about it, we actually rely on dask[dataframe] instead of dask, and this will also include ffspec automatically. I like this solution more - because currently people installing our package might have problems with that also (not only in the tests).